cluster_kwargs.yaml

```diff
@@ -31,6 +31,12 @@ parquet_cluster:
   n_workers: 15
   worker_vm_types: [m5.xlarge]  # 4 CPU, 16 GiB
 
+# For tests/workflows/test_embarrassingly_parallel.py
+embarrassingly_parallel_cluster:
+  n_workers: 100
+  backend_options:
```

Comment:
Is there a reason we are not specifying the VM types here? Using the default t3 will cause a lot of volatility in the times and trigger regressions. I'd recommend using `m6i.xlarge` and that will take ~4 min total

Reply:
I want this to be representative of what a naive user might do. My current thinking is that they'll just use the default worker specification. Though maybe you or someone else have a different perspective. Definitely open to thoughts from others

Reply:
It makes sense, although this will likely cause regression detections. I was able to see a difference of double the time between a run that Mat did and one I did with the defaults. 
Maybe @ntabris has some opinions on this?